dev-docs/bidders/ix.md

```diff
@@ -11,6 +11,7 @@ schain_supported: true
 coppa_supported: true
 tcfeu_supported: true
 floors_supported: true
+fledgeForGpt_supported: true
```

Comment:
Sorry... rough day. I meant to remove this one and then I can merge this and I'll keep you posted on what we settle